forEach/challenges-01.test.js

```diff
@@ -0,0 +1,356 @@
+"use strict";
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named pushIntoSecond which takes in an array and uses a for loop
+// to push all of the elements from the initial array into a second array.
+//
+// Return the second array.
+// ------------------------------------------------------------------------------------------------
+
+const pushIntoSecond = arr => {
+  const second = [];
+  // Solution code here...
+  for (let i = 0, l = arr.length; i < l; i++) {
```

Comment:
making variable `l` is unnecessary, just do `for (let i = 0; i < arr.length; i++)`